frontend/src/components/Notificacao/Evolucao/Cons.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div>
+    <v-row>
+      <v-col>
+        <h4 class="primary--text font-weight-medium title">IDENTIFICAÇÃO DO NOTIFICANTE</h4>
+        <v-row dense>
+          <v-col>
+            <strong>Nome</strong>
+            <p>{{notificacao.nome}}</p>
+          </v-col>
+          <v-col>
+            <strong>Documento</strong>
+            <p>{{notificacao.documento}}</p>
+          </v-col>
+          <v-col>
+            <strong>Telefone</strong>
+            <p>{{notificacao.telefone}}</p>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col>
+        <v-data-table
+          v-model="selected"
+          :headers="headers"
+          :items="evolucoes"
+          item-key="id"
+          :options.sync="options"
+          :server-items-length="totalEvolucoes"
+          :loading="loading"
+          loading-text="Carregando as evoluções do paciente."
+          no-data-text="Não há evoluções até o momento."
+          :footer-props="{
+            itemsPerPageText: 'Linhas por página'
+          }"
+          class="elevation-1"
+        >
+          <template v-slot:top>
+            <v-card-title>Listagem das Evoluções do Paciente</v-card-title>
+          </template>
+        </v-data-table>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+<script>
+import NotificacaoService from '@/services/NotificacaoService';
+import NotificacaoEvolucao from '@/entities/NotificacaoEvolucao';
+
+const EVOLUCOES = [
+  {
+    id: '1',
+    dtEvolucao: '2020-04-18T15:53:01.553Z',
+    tpLocal: 'Alta com isolamento domiciliar',
+    tpEvolucao: 'Suspeito',
+  },
+  {
+    id: '2',
+    dtEvolucao: '2020-04-18T15:53:01.553Z',
+    tpLocal: 'Hospitalizado – Leito comum',
+    tpEvolucao: 'Confirmado',
+  },
+  {
+    id: '3',
+    dtEvolucao: '2020-04-18T15:53:01.553Z',
+    tpLocal: 'Hospitalizado – Leito comum',
+    tpEvolucao: 'Curado',
+  },
+  {
+    id: '4',
+    dtEvolucao: '2020-04-18T15:53:01.553Z',
+    tpLocal: 'Alta com isolamento domiciliar',
+    tpEvolucao: 'Encerrado',
+  },
+  {
+    id: '5',
+    dtEvolucao: '2020-04-18T15:53:01.553Z',
+    tpLocal: 'Alta com isolamento domiciliar',
+    tpEvolucao: 'Descartado',
+  },
+  {
+    id: '6',
+    dtEvolucao: '2020-04-18T15:53:01.553Z',
+    tpLocal: 'Hospitalizado - Leito UTI',
+    tpEvolucao: 'Óbito',
+  },
+];
+
+export default {
+  data: () => ({
+    selected: [],
+    loading: true,
+    options: {},
+    totalEvolucoes: 0,
+    evolucoes: [],
+    notificacao: {},
+    headers: [
+      { text: 'Data da atualização', value: 'dataHoraAtualizacao' },
+      { text: 'Local', value: 'local' },
+      { text: 'Situação', value: 'situacao' },
+    ],
+  }),
+  watch: {
```

Comment:
Watch não é muito legal usar, use somente em último caso.
Veja se o v-data-table não emite nenhum evento ao alterar o valor de options, para que você capture esse evento e chame a consultarNotificacoes sem precisar do watch

Reply:
Na documentação pede pra usar isso, pq ao navegar nas paginas ele automaticamente, emite alteração no options ai neste caso preciso paginar novamente.

Reply:
A ideia do modificador .sync é atualizar todos os objetos, sem necessidade de um evento para cada um do componente, é pra facilitar pra você não precisar criar uma função pra cada campo do objeto e somente atualizar o objeto dentro dessas função.
Mas a partir do momento que alterando essa propriedade, precisamos fazer algo além de atualizar no objeto (no caso, fazer uma chamada http), o ideal é que capturemos o evento, em vez de forçar um watch, que é menos legível, e usa mais recursos.
Nesse caso, vamos precisar recarregar os registros quando **todas** os atributos de options forem alterados?
Na documentação desse componente, tem a parte de Events, dá uma olhada lá nos eventos que ele emite. Os que você vai precisar são os update:algumaCoisa